### PR TITLE
Add Python style guide

### PR DESCRIPTION
**Pull request by ljiang09**

### CODE REVIEW

File: style_guides/PYTHON_STYLEGUIDE.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+## Docstrings
+
+Each file, function, and class should have a docstring. This should be of the
+format specified in
+https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings,
+surrounded by three quotes and occurring inside the file/function/class but
+before any corresponding code.
+
+Example:
```
Comment (aditivinod) [inline style_guides/PYTHON_STYLEGUIDE.md]: Nice example

File: style_guides/PYTHON_STYLEGUIDE.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- Indentations should be 4 spaces
+- All functions and files should be documented with a docstring
+- All lines should be 80 characters or less
+
+## Naming Conventions
+
+Use MACRO_CASE for global constants, snake_case for everything else (variables,
+functions, etc)
+
+Names can only be a single character in the following cases:
```
Comment (aditivinod) [inline style_guides/PYTHON_STYLEGUIDE.md:80]: Good exceptions/setting standards

**Review by aditivinod — CHANGES_REQUESTED** [review]
Looks great overall - one minor change for both: 
Arguments/return type specifications are shown clearly in the example, but it might also be helpful to include a single sentence about those.

**Comment by ljiang09** [comment]
@aditivinod thanks for the comments!

**Review by aditivinod — APPROVED** [review]
LGTM (:

**Pull request merged**